cli/azd/internal/appdetect/java.go

```diff
@@ -219,35 +242,49 @@ func detectDependencies(currentRoot *mavenProject, mavenProject *mavenProject, p
 				}
 				if !contains(destinations, destination) {
 					destinations = append(destinations, destination)
-					log.Printf("Event Hubs [%s] found for binding [%s]", destination, bindingName)
```

Comment:
Move log to one place, which can help to avoid this: When update log, updated one place and forget another place.